content/posts/January-20.md

```diff
@@ -46,3 +46,17 @@ draft: false
 - Reports invalid or missing packit config. ([#328](https://github.com/packit-service/packit-service/pull/328))
 - Status workflow has been changed and simplified. ([#338](https://github.com/packit-service/packit-service/pull/338))
 - Minor fixes. ([#323](https://github.com/packit-service/packit-service/pull/323), [#326](https://github.com/packit-service/packit-service/pull/326), [#329](https://github.com/packit-service/packit-service/pull/329), [#331](https://github.com/packit-service/packit-service/pull/331), [#335](https://github.com/packit-service/packit-service/pull/335), [#336](https://github.com/packit-service/packit-service/pull/336))
+
+## Week 4
+
+### packit
+
+- - [0.8.1](https://pypi.org/project/packitos/0.8.1/) has been released.
+- - CLI has bash auto completion. ([#654](https://github.com/packit-service/packit/pull/654))
+- - Few bugs have been squashed. ([#668](https://github.com/packit-service/packit/pull/668), [#670](https://github.com/packit-service/packit/pull/670), [#676](https://github.com/packit-service/packit/pull/676), [#677](https://github.com/packit-service/packit/pull/677), [#678](https://github.com/packit-service/packit/pull/678) )
```

Comment:
The double `-` are here for some purpose?

Reply:
ajaj copy pasta